build.gradle

```diff
@@ -196,6 +196,7 @@ allprojects {
         }
 
         systemProperty 'junit.jupiter.execution.timeout.default', junitMethodExecutionTimeout
+        systemProperty 'kotlinx.coroutines.test.default_timeout', junitMethodExecutionTimeout
```

Comment:
the property name being "JunitMethodExecutionTimeout" is a bit misleading now that it's controlling non-junit things, but it seems dumb to _not_ have these be the same timeout :shrug: